http/logical.go

```diff
@@ -482,7 +497,9 @@ WRITE_RESPONSE:
 	if cacheControl, ok := resp.Data[logical.HTTPRawCacheControl].(string); ok {
```

Comment:
Should we consider pulling this Cache-Control header into the set of headers that we allow to be customized as well? The default `Cache-Control: no-store` (set at https://github.com/hashicorp/vault/blob/main/http/handler.go#L303-L305) aligns with current Mozilla guidance (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#preventing_caching) and may be acceptable to many Vault operators, but some may prefer different values depending on security requirements or what vulnerability scanner they're trying to appease.

Reply:
Yes, indeed it is customizable. Couple of lines down (line 501) will allow setting that if operators included that into the config file.